src/HealthChecks.AzureServiceBus/Configuration/AzureEventHubOptions.cs

```diff
@@ -0,0 +1,41 @@
+using Azure.Core;
+using Azure.Messaging.EventHubs;
+
+namespace HealthChecks.AzureServiceBus.Configuration;
+
+/// <summary>
+/// Azure Event Hub configuration options.
```

Comment:
```suggestion
/// Configuration options for <see cref="AzureEventHubHealthCheck"/>.
```

By convention as well.